--- labyrinth-2.c	2024-11-22 23:28:22.526426102 +0300
+++ labyrinth-3.c	2024-11-22 23:30:07.296406462 +0300
@@ -10,14 +10,14 @@
 #define DOWN 4
 
 int main(int argc, char **argv) {
-    if (argc != 3 || strlen(argv[1]) != 2) {
-        printf("Please pass string of 2 characters (space and wall) and number (size of labyrinth)\n");
+    if (argc != 4 || strlen(argv[2]) != 2) {
+        printf("Please pass number (init value of random generator), string of 2 characters (space and wall) and number (size of labyrinth)\n");
         return 1;
     }
-    const char wallChar = argv[1][0], spaceChar = argv[1][1];
-    const int size = atoi(argv[2]);
+    const char wallChar = argv[2][0], spaceChar = argv[2][1];
+    const int size = atoi(argv[3]);
 
-    srand(time(NULL));
+    srand(atoi(argv[1]));
 
     bool **points = malloc((2*size+1) * sizeof(bool *));
 
